README.md

```diff
@@ -47,6 +47,26 @@ You can use Alfred to
 
 `command + d` : when given "don't save, cancel, and save", triggers "don't save"; also short for saving something to the desktop
 
+## Terminal / Bash
+
+`ctrl + a` : jump to the beginning of line
+
+`ctrl + u` : clear line
+
+`↑` and `↓` : cycle through command history 
+
+`!!` : last command, e.g. `sudo !!`, `!! > log.txt`
+
+`|` : piping warrants its own section
+
+`alias | grep` : search through aliases
+
+`pbcopy` and `pbpaste` : copy to and paste from the clipboard
+
+`pwd` : print (show) current working directory
+
+`pushd` and `popd` : save and retrieve the current working directory
```

Comment:
Please break these into separate commands and include mention of the path arguments that they take.